test/OpenTelemetry.Resources.AWS.Tests/AWSEC2DetectorTests.cs

```diff
@@ -7,6 +7,18 @@ namespace OpenTelemetry.Resources.AWS.Tests;
 
 public class AWSEC2DetectorTests
 {
+    private static class ExpectedSemanticConventions
+    {
+        public const string AttributeCloudProvider = "cloud.provider";
+        public const string AttributeCloudPlatform = "cloud.platform";
+        public const string AttributeCloudAccountID = "cloud.account.id";
+        public const string AttributeCloudAvailabilityZone = "cloud.availability_zone";
+        public const string AttributeCloudRegion = "cloud.region";
+        public const string AttributeHostID = "host.id";
+        public const string AttributeHostType = "host.type";
+        public const string AttributeHostName = "host.name";
+    }
```

Comment:
see if we can get rid of this and use defined SemanticConventios to ensure if something changed, the test will fail.

Reply:
commented elsewhere